Floofbot/Migrations/FloofDataContextModelSnapshot.cs

```diff
@@ -0,0 +1,113 @@
+
+                    b.Property<ulong>("MessageDeletedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("MessageUpdatedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("ServerId")
+                        .HasColumnName("ServerID")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserBannedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserJoinedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserKickedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserLeftChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserMutedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserUnbannedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("UserUnmutedChannel")
+                        .HasColumnType("INTEGER");
+
+                    b.ToTable("LogConfigs");
+                });
+
+            modelBuilder.Entity("Floofbot.Services.Repository.Models.Tag", b =>
+                {
+                    b.Property<ulong>("TagId")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnName("TagID")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("Content")
+                        .HasColumnType("TEXT");
+
+                    b.Property<ulong>("UserId")
+                        .HasColumnName("UserID")
+                        .HasColumnType("INTEGER");
+
+                    b.HasKey("TagId");
+
+                    b.ToTable("Tags");
+                });
+
+            modelBuilder.Entity("Floofbot.Services.Repository.Models.Warning", b =>
+                {
+                    b.Property<ulong>("Id")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<DateTime>("DateAdded")
+                        .HasColumnType("TEXT");
+
+                    b.Property<bool>("Forgiven")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("ForgivenBy")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("GuildId")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<ulong>("Moderator")
+                        .HasColumnType("INTEGER");
+
+                    b.Property<string>("Reason")
+                        .HasColumnType("TEXT");
+
+                    b.Property<ulong>("UserId")
+                        .HasColumnType("INTEGER");
+
+                    b.HasKey("Id");
+
+                    b.ToTable("Warnings");
+                });
```

Comment:
Is this a different schema from nadeko? iirc it was to remain the same so that warnings can be moved over easily